src/fides/api/models/privacy_experience.py

```diff
@@ -129,6 +129,7 @@ class ExperienceTranslationBase:
     privacy_policy_link_label = Column(String)
     privacy_policy_url = Column(String)
     privacy_preferences_link_label = Column(String)
+    modal_link_label = Column(String)
```

Comment:
Do our out of the box experience configs in plus need to ship with this value?

Reply:
nope, we set a default internally in fides-js so no need to have this db-side.

Reply:
https://github.com/ethyca/fides/pull/4766/files#diff-73f3ee3f607d9c4cb47cd9edc7aeda7afc47df7b506ebca6f9f714efcb71b6a1R634

Reply:
I don't understand. Is this not something we will _ever_ supply out of the box translations for? Why is this field different?

Reply:
Hmm I see, OOB translations might be handy in the future. I'd like to consult with @mfbrown on this one.

To answer your Q, this field is diff from the other fields on experience translation in a couple ways:
1. It's not used automatically by fides internally to render anything in the UI. It's only used when the customer writes js on their site to explicitly grab this value from fides, e.g. `Fides.getModalLinkLabel()`
2. Since this field is used to actually add/replace text on a customers' site, it's possible we might want more predictability with only returning something they've explicitly configured rather than an OOB translation, but I'd like @mfbrown to confirm.

Reply:
plus PR here, to address having OOB translations for this field in the future- https://github.com/ethyca/fidesplus/pull/1376